packages/transformers/js/src/JSTransformer.js

```diff
@@ -251,17 +252,58 @@ export default (new Transformer({
       scope_hoist: asset.env.shouldScopeHoist,
     });
 
-    let convertLoc = loc => ({
-      filePath: relativePath,
-      start: {
-        line: loc.start_line,
-        column: loc.start_col,
-      },
-      end: {
-        line: loc.end_line,
-        column: loc.end_col,
-      },
-    });
+    let convertLoc = loc => {
```

Comment:
We should probably extract this into a util or even make part of core as it's quite a bit of logic